logos/logos-ui/public/claude-logos.ps1

```
@@ -0,0 +1,428 @@
+# 20000 tokens of context for nothing.
+$ContextForCli = $ContextTokens - $Headroom
+$CompactAt = $ContextForCli - $MaxOutputTokens - 13000
+$HardStopAt = $ContextForCli - $MaxOutputTokens - 3000
+
+function Write-ContextReport {
+    Write-Host ("model    : {0}" -f $LogosModel)
+    Write-Host ("logos    : {0}" -f $LogosUrl)
+    if ($ContextOrigin -eq 'estimate') {
+        Write-Host ("context  : {0:N0} tokens (an estimate — Logos reports no size for this model)" -f $ContextTokens)
+    } else {
+        Write-Host ("context  : {0:N0} tokens, using \"{1}\" of what Logos offers" -f $ContextTokens, $ContextOrigin)
```

PSScriptAnalyzer (1.25.0)

[error] Unexpected token '{' in expression or statement.

(UnexpectedToken)